config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3061,6 +3061,10 @@
 # DEFAULT: 500
 # site.entity.pagesize.maximum=200
 
+# Set the site entity broker properties that can be updated by a maintain type user.
+# DEFAULT: none (null)
```

Comment:
Is there any reason why `contact-email` and `contact-name` shouldn't be the default values?